crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -367,13 +378,16 @@ impl<'a> Visitor<'a> {
         };
 
         let mut logger = self.sink.logger(behavior);
-        if self.run_opts.is_github_actions {
-            let package = if self.run_opts.single_package {
-                None
+        if let Some(vendor_behavior) = vendor_behavior {
+            let group_name = if self.run_opts.single_package {
+                ""
             } else {
-                Some(task_id.package())
+                task_id.package()
             };
```

Comment:
We to include both package name and task name in the group name if it's a multi-package repo and the task name if it is just single package mode. This matches previous behavior.

e.g.
```
let group_name = if self.run_opts.single_package {
    task_id.task().to_string()
} else {
    format!("{}:{}", task_id.package(), task_id.task())
}
```